lib/tool/code_interpreter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Tool
+  class CodeInterpreter < Base
+    description <<~DESC
```

Comment:
Would you please write a description to this Tool (code comments) like what this tool does, how to use it, etc. Thank you!